src/rules/control-has-associated-label.js

```diff
@@ -21,6 +21,8 @@ import mayHaveAccessibleLabel from '../util/mayHaveAccessibleLabel';
 
 const errorMessage = 'A control must be associated with a text label.';
 
+const blacklist = ['link'];
```

Comment:
```suggestion
const ignoreList = ['link'];
```
although why not make this a Set?